ftdetect/gentoo.vim

```diff
@@ -31,20 +31,20 @@ au BufNewFile,BufRead ChangeLog*
 
 " /etc/init.d/ scripts
 au BufNewFile,BufRead /etc/init.d/*
-    \     set filetype=gentoo-init-d |
+    \     set filetype=gentoo-init-d.sh |
```

Comment:
I've never seen that. Could you point me to the relevant docs?

Reply:
`:help 'filetype'`

> When a dot appears in the value then this separates two filetype names.

Reply:
Oh nice, now I know how to look up options without getting .txt files in the way.